js/libs/core.js

```diff
@@ -353,6 +357,14 @@ Fliplet.FormBuilder = (function() {
         type: Array
       };
 
+      component.props._browserSupport = {
```

Comment:
why not simply creating a instance method that is able to check if a browser is a certain one?

e.g.

```
function isBrowser(name) {
  // create a switch case, not an if condition
  if (name === 'IE11') {
    return navigator.userAgent.indexOf('Trident/') !== -1;
  }
}
```